doc/release-notes/release-notes-0.11.2.z9.md

```diff
@@ -19,37 +20,38 @@ Jack Grigg (11):
       Add separate lock for SpendingKey key store operations.
       Test conversion between solution indices and minimal representation.
 
-Daira Hopwood (6):
+Daira Hopwood:
       Move bigint arithmetic implementations to libsnark.
       Add mostly-static checks on consistency of Equihash parameters, MAX_HEADERS_RESULTS, and MAX_PROTOCOL_MESSAGE_LENGTH.
       Change some asserts in equihash.cpp to be static.
       Decrease MAX_HEADERS_RESULTS to 160. fixes #1289
       Increment version numbers for z9 release.
       Add these release notes for z9.
 
-Taylor Hornby (5):
+Taylor Hornby:
       Disable hardening when building for coverage reports.
       Upgrade libsodium for AVX2-detection bugfix.
       Fix inconsistent optimization flags; single source of truth.
       Add -fwrapv -fno-strict-aliasing; fix libzcash flags.
       Use libsodium's s < L check, instead checking that libsodium checks that.
 
-Simon Liu (3):
+Simon Liu:
       Fixes #1193 so that during verification benchmarking it does not unncessarily create thousands of CTransaction objects.
       Closes #701 by adding documentation about the Payment RPC interface.
       Add note about zkey and encrypted wallets.
 
-Gaurav Rana (1):
+Gaurav Rana:
       Update zcash-cli stop message.
 
-Tom Ritter (1):
+Tom Ritter:
       Clarify comment about nonce space for Note Encryption.
 
-Robert C. Seacord (1):
+Robert C. Seacord:
       Memory safety and correctness fixes found in NCC audit.
 
-Patrick Strateman (1):
+Patrick Strateman (merged by Taylor Hornby):
```

Comment:
Tested locally, and merging these into `Patrick Strateman (1) (merged by Taylor Hornby):` results in the author line `Patrick Strateman (merged by Taylor Hornby) (1)`